Places/Scenes/CreatePlace/CreatePlaceViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  CreatePlaceViewController.swift
+//  Places
+//
+//  Created by Andressa Valengo on 24/01/25.
+//  Copyright (c) 2025 ___ORGANIZATIONNAME___. All rights reserved.
+//
+
+import UIKit
+
+protocol CreatePlaceDisplayLogic: AnyObject {
+  func displayCreatedPlace(viewModel: CreatePlace.CreatePlace.ViewModel)
+  func displayPlaceToEdit(viewModel: CreatePlace.EditPlace.ViewModel)
+  func displayUpdatedPlace(viewModel: CreatePlace.UpdatePlace.ViewModel)
+}
+
+class CreatePlaceViewController: UIViewController, CreatePlaceDisplayLogic {
```

Comment:
```suggestion
final class CreatePlaceViewController: UIViewController, CreatePlaceDisplayLogic {
```
Essa classe não será (ou não deveria ser) herdada por nenhuma outra classe, adicionar o final dá uma leve otimizada na compilação.